wait-for-stack-teardown.bash

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
What about some better place for this script than root src directory ? Put it in to `utils` dir ?

Reply:
I've put it with the workflow files, where it's being called from.

Reply:
strange, then GH is not showing full file name path to me